coremltools/converters/mil/frontend/tensorflow/test/test_ops.py

```diff
@@ -5616,7 +5616,7 @@ def test_top_k(self, compute_unit, backend, rank, k, sort):
             pytest.skip("iOS16 version topk needed for sort = False")
         if not sort and _macos_version() < (13, 0):
             pytest.skip("New functionality in macOS13/iOS16")
-        if backend[0] == "neuralnetwork" and rank == 5 and k is None and sort:
```

Comment:
can we please guard this by platform.machine*() instead?

Reply:
I was thinking we might should guard this entire `rank == 5 and k is None and sort` case?

Months back I found neuralnetwork backend failing on arm machine (weird why mlprogram not), then today I found mlprogram backend failing on intel machine